### DISCUSSION TITLE
Question about threshold.conf and disabledsid.conf

### DISCUSSION THREAD

**Discussion by jdpadro** [Security-Onion-Solutions/securityonion#4539] [Unsupported Versions]
When moving away from 16.04.6.1 I copied both tthe threshold.conf and disabledsid.conf files to my jump box.

Threshold
I did see your instructions about using a pillar for the threshold. While looking withint the /opt/so/saltstack/local/pillar/ I see a directory called thresholding. Looking within this directory I see no sls file there. Is this where I would create a global threshold pillar to push the same suppressions across all sensors? If so what should that file be named? 

Disable that SID
I see that I can use "so-rule disable [SID] to disable those pesky SID;s. When using the previous method you were required to provide the gen_id "1:123456789" to ensure that it properly disabled that particular SID. Is this no longer required within SO2 and using so-rule?
Thanks

**Comment by weslambert** [comment]
Hi @jdpadro ,

In order to apply a threshold globally, you would add the threshold to `/opt/so/saltstack/local/pillar/global.sls`.
(https://docs.securityonion.net/en/latest/managing-alerts.html#threshold)

You can still do this manually, either in the relevant minion pillar (`/opt/so/saltstack/local/pillar/minions/<minionid>.sls` -- for only a single node), or via the manager's pillar in the `idstools` section (`/opt/so/saltstack/local/pillar/minions/<manager>.sls`).

Ex.

```
idstools:
  sids:
    disabled:
      - '1:2033064'
```

Then run `salt-call state.apply idstools`.
